screens/Contacts/ContactListScreen.tsx

```diff
@@ -167,7 +169,9 @@ export const ContactListScreen = () => {
   }
 
   const RenderOneContact = ({ item }: { item: Contact }) => (
-    <View style={styles.cardContainer}>
+    <TouchableOpacity 
+      style={styles.cardContainer}
+      onPress={() => useNav.navigate({name: "ExternalProfile", params: {uid: item.uid}} as never)}>
```

Comment:
Why are you using `as never`?

I think that `never` is supposed to be used in very precise situations, this one not being one of them.

[See here for more information](https://stackoverflow.com/questions/72001800/can-someone-explain-to-me-what-does-as-never-do-in-this-piece-of-code)

Reply:
You're right, I didn't look into it but just replicated some existing code. I modified it. Note that we are using as never quite a lot when navigating and I guess this is a mistake.

<img width="319" alt="Capture" src="https://github.com/uniconnect-epfl/uniconnect/assets/93329823/65c3f453-86c7-4446-b26e-48670e986d66">

Reply:
Yes, I think that here people use it for type inference but worst case scenario you should use `as any`.